apps/dapp/src/providers/SwapProvider.tsx

```diff
@@ -110,48 +172,51 @@ export const SwapProvider = (props: PropsWithChildren<{}>) => {
     setError(null);
 
     const tokenAddress = token === TICKER_SYMBOL.FEI ? env.contracts.fei : env.contracts.frax;
-    const ammRouter = new TempleStableAMMRouter__factory(signer).attach(env.contracts.templeV2Router);
     const tokenContract = new ERC20__factory(signer).attach(tokenAddress);
-
     const balance = await tokenContract.balanceOf(wallet);
     const verifiedAmountIn = amountIn.lt(balance) ? amountIn : balance;
 
-    const deadlineInSeconds = deadlineInMinutes * 60;
-    const deadline = formatNumberFixedDecimals(Date.now() / 1000 + deadlineInSeconds, 0);
-
-    await ensureAllowance(token, tokenContract, env.contracts.templeV2Router, amountIn);
-
+    let receipt: TransactionReceipt | undefined;
     try {
-      const buyTXN = await ammRouter.swapExactStableForTemple(
-        verifiedAmountIn,
-        minAmountOutTemple,
-        tokenAddress,
-        wallet,
-        deadline,
-        {
-          gasLimit: env.gas?.swapFraxForTemple || 300000,
-        }
-      );
-      const txReceipt = await buyTXN.wait();
-
-      AnalyticsService.captureEvent(AnalyticsEvent.Trade.Buy, { token, amount: formatBigNumber(verifiedAmountIn) });
-
-      // Show feedback to user
-      openNotification({
-        title: `Sacrificed ${token}`,
-        hash: buyTXN.hash,
-      });
-
-      return txReceipt;
+      const nonFrax = [TICKER_SYMBOL.USDC, TICKER_SYMBOL.USDT, TICKER_SYMBOL.DAI];
```

Comment:
We decided we're going to rug the old frax AMM altogether, so we can fully rely on 1inch for all swaps